Packs/CommonTypes/IndicatorFields/incidentfield-detections.json

```
@@ -0,0 +1,59 @@
+{
+    "id": "indicator_detections",
+    "version": -1,
+    "modified": "2023-04-19T12:09:09.282884667Z",
+    "name": "Detections",
+    "ownerOnly": false,
+    "cliName": "detections",
+    "type": "grid",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
+    "unmapped": false,
+    "unsearchable": true,
+    "caseInsensitive": true,
+    "columns": [
+        {
+            "key": "type",
+            "displayName": "Type",
+            "type": "shortText",
+            "orgType": "shortText",
+            "required": false,
+            "script": "",
+            "width": 100,
+            "isDefault": true,
+            "fieldCalcScript": "",
+            "isReadOnly": false,
+            "selectValues": null
+        },
+        {
+            "key": "content",
+            "displayName": "Content",
+            "type": "url",
```

```suggestion
            "type": "shortText",
```

My bad, changing back, sorry :(